installer.sh

```diff
@@ -111,3 +113,20 @@ if ! [[ -f "external/objdump" ]] ; then
 else
   echo -e "$ORANGE""objdump is already downloaded and compiled""$NC"
 fi
+
+# aha for html generation
+echo -e "\\n""$ORANGE""$BOLD""Downloading aha""$NC"
```

Comment:
Your code won't work at my system. I tried it with some little changes, please fix it :) 

```
# aha for html generation
echo -e "\\n""$ORANGE""$BOLD""Downloading aha""$NC"
if ! [[ -f "external/aha-master" ]] ; then
  apt-get install make
  apt-get install unzip
  wget https://github.com/theZiz/aha/archive/master.zip -O external/aha-master.zip
  unzip ./external/aha-master.zip -d ./external
  cd ./external/aha-master || exit 1
  echo -e "$ORANGE""$BOLD""Compile aha""$NC"
  make
  cd ../.. || exit 1
  mv "external/aha-master/aha" "external/aha"
  rm -R external/aha-master
else
  echo -e "$ORANGE""aha is already downloaded and compiled""$NC"
fi
```

I don't know, if `aha` needs stuff from the zip archive. I deleted the unzipped archive to keep the external directory clean. If it won't work, change it.

Reply:
Thank you, the changes are fine. 
Furthermore I have added rm ./external/aha-master.zip to remove the zip file. Now everything should be clean.